src/App.js

```diff
@@ -1,9 +1,30 @@
-function App() {
+import { useEffect } from 'react';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { fetchCountries } from './redux/countries/countries';
+import { fetchCovidData } from './redux/covid-data/covidData';
+import Home from './components/Home';
+import CountryCard from './components/view/country/CountryCard';
+import AllCards from './components/view/region/RegionCards';
+
+const App = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchCountries());
+    dispatch(fetchCovidData());
+  }, []);
+
   return (
     <>
```

Comment:
- [ ] The home page is working very fine and the details page also but it is difficult for me to move from the details page back to the home page without changing the URL. Please can you add a homepage link that will send us back to the homepage directly? It will be very helpful

Reply:
Will do so. Thanks for the suggestion.